guides/common/modules/proc_using-ansible-job-templates-for-package-and-errata-actions.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Using_Ansible_Job_Templates_for_Package_and_Errata_Actions{context}"]
+= Using Ansible job templates for package and errata actions
+
+By default, {Project} is configured to use the Script provider templates for remote execution jobs.
+If you prefer using the Ansible provider templates for your remote jobs, you can configure {Project} to use the Ansible provider templates for remote execution features that take a search query input.
```

Comment:
This reads to me a little bit as if "{Project} will always use Script for *ALL* rex jobs. If I follow these steps it will use ansible for *ALL* jobs".

Reply:
@jeremylenz Thoughts on how this should be written?